src/main/scala/EnsimePlugin.scala

```diff
@@ -29,6 +30,16 @@ object EnsimeKeys {
     "Arguments for the java presentation compiler, extracted from the compiler flags."
   )
 
+  val ensimeIgnoreSourcesInBase = settingKey[Boolean](
+    "ENSIME doesn't support sources in base, this tolerates their existence."
+  )
+  val ensimeIgnoreMissingDirectories = settingKey[Boolean](
+    "ENSIME requires declared source directories to exist. If you find this noisy, ignore them."
+  )
+  val ensimeIgnoreScalaMismatch = settingKey[Boolean](
+    "ENSIME can only use one version of scala per project. If you have mixed scala versions (e.g. lib and sbt plugin), set this to acknowledge."
```

Comment:
"per project" is misleading. Maybe go for something like "for the entire build" or "for the build definition".